client/ayon_deadline/abstract_submit_deadline.py

```diff
@@ -233,6 +233,9 @@ def get_generic_job_info(self, instance: pyblish.api.Instance):
         if job_info.SecondaryPool != "none":
             job_info.SecondaryPool = job_info.SecondaryPool
 
+        if job_info.Frames:
```

Comment:
I might be wrong, but `Frames` is requirement for job info, it does not mean it was filled by artist...

Reply:
DL's `Frames` are filled always, `hasExplicitFrames` is denoting that custom frames were used. Without it I would need to somehow figure out that content of `Frames` is not default range.

`Frames` is Deadline functionality, we tested multiple DCCs where it is supported (AE, Nuke, Maya, Max, Houdini).